src/components/Logged.vue

```diff
@@ -124,9 +125,29 @@ export default {
       email: '',
       logged: false,
       inLocal: false,
+      hidden: true,
     };
   },
   methods: {
+    hideNav() {
+      if (!this.hidden) {
```

Comment:
Can you handle this with v-if or v-show properties.
If it's not in same component, you can use Vuex state to communicate between those components.
This is pretty much "Do not use unless it's really necessary" approach

Reply:
https://github.com/nsftx/incubator-2019-rating-app-admin-client/pull/60/commits/6e3f5f29e739a6b16d7982c4bfac82fc9762c46e

Reply:
https://github.com/nsftx/incubator-2019-rating-app-admin-client/pull/60/commits/402c63cf330f107cc18f5e36dbd294a5be94b6bc